src/assets/scss/FloatingActionButton.scss

```diff
@@ -0,0 +1,49 @@
+.parentfab {
+  position: relative
+}
+
+.fab {
+  width: 7.0rem;
+  height: 7.0rem;
+  background-color: #dc8905;
+  border-radius: 50%;
+  transition: all 0.1s ease-in-out;
```

Comment:
Use `transition: transform 0.1s ease-in-out;` since the only property you are applying the transition on is `transform`. This is important for performance optimization.